zinc/lib/zinc_interpreter/Zinc_interpreter.ml

```diff
@@ -43,12 +43,13 @@ module Make (D : Domain_types) = struct
     let[@warning "-4"] eval (module E : Executor) (code, env, stack) =
       let apply_once (code : Zinc.t) env stack =
         let () =
-          print_endline
-            (Format.asprintf
-               "interpreting:\ncode:  %s\nenv:   %s\nstack: %s"
-               (Zinc.to_string code)
-               (Env.to_string env)
-               (Stack.to_string stack))
+          (fun _ -> ())
+          @@ lazy
```

Comment:
Can you explain the benefit of doing this?

Reply:
oh, my bad this is from unrelated pr im working on, removed this